CHANGELOG.md

```diff
@@ -19,6 +19,22 @@ Twilio Voice React Native SDK has now reached milestone `beta.5`. Included in th
   const millisecondsSinceEpoch = date.getTime();
   ```
 
+- The API for Call Invite events has now changed.
+
+  The following events have been removed from the `Voice` class:
+  - `Voice#callInviteAccepted`
+  - `Voice#callInviteRejected`
+  - `Voice#callInviteNotificationTapped`
+  - `Voice#cancelledCallInvite`
+  Please see the `Voice` class API documentation [here](https://github.com/twilio/twilio-voice-react-native/blob/latest/docs/api/voice-react-native-sdk.voice_class.md) for details.
+
+  The following events have been added to the `CallInvite` class:
+  - `CallInvite#accepted`
+  - `CallInvite#rejected`
+  - `CallInvite#notificationTapped`
+  - `CallInvite#cancelled`
+  Please see the `CallInvite` class API documentation [here](https://github.com/twilio/twilio-voice-react-native/blob/latest/docs/api/voice-react-native-sdk.callinvite_class.md) for details.
```

Comment:
Can we combine these instead of two separate blocks? Something like "the following events has been moved from Voice to CallInvite..."
- Voice#callInviteAccepted -> CallInvite#accepted
- ...